sources/deny.toml

```diff
@@ -58,17 +58,15 @@ wildcards = "deny"
 
 deny = [
     { name = "structopt" },
-    { name = "clap", wrappers = [ "cargo-readme" ] },
+    { name = "clap", wrappers = ["cargo-readme"] },
 ]
 
 skip = [
-    # several dependencies use an older version of toml
-    { name = "toml", version = "=0.5" },
-    # tungstenite uses a newer version of base64, but many of the
-    # first party crates are still in this version
-    { name = "base64", version = "=0.13" },
-    # AWS SDK for rust uses an older version of fastrand
-    { name = "fastrand", version = "=1.9" },
+    # sdk still uses 0.7.5 but everything in this repo now uses 0.8.2
+    # this can be removed once sdk is updated to 0.8.2
+    { name = "heck", version = "=0.4.1" },
+    { name = "snafu", version = "=0.7.5" },
+    { name = "snafu-derive", version = "=0.7.5" },
```

Comment:
Can we have comments for each one just to be clear which ones go away when? If things get refactored/moved I want them to keep their context going forward.

Reply:
They all can go away once the above comment is done (bottlerocket-sdk) they are all from snafu